marimo/_output/formatters/formatters.py

```diff
@@ -149,7 +152,13 @@ def find_spec(  # type:ignore[no-untyped-def]
             original_find_spec=original_find_spec,
         ) -> Any:
             del self
-            spec = original_find_spec(fullname, path, target)
+
+            try:
+                spec = original_find_spec(fullname, path, target)
+            except Exception as e:
+                LOGGER.warning(f"Error finding spec for {fullname}: {e}")
```

Comment:
I see, I guess the question is why is this getting patched in in the first place?